srtcore/sync.h

```diff
@@ -460,14 +454,14 @@ inline void releaseCond(Condition& cv) { cv.destroy(); }
 class CSync
 {
     Condition* m_cond;
-    CGuard* m_locker;
+    UniqueLock* m_locker;
 
 public:
-    // Locked version: must be declared only after the declaration of CGuard,
+    // Locked version: must be declared only after the declaration of ScopedLock,
```

Comment:
Here you should mentioned in the comment the same type that was used in the constructor parameters - `UniqueLock`

Reply:
Fixed.